teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md

```diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -ConnectorId
-The connector id
+### -ConnectionId
+Gets or sets the WFM connection id for the new instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```

Comment:
```suggestion
Gets or sets the WFM connection ID for the new instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```